.github/workflows/test.yaml

```diff
@@ -50,37 +48,62 @@ jobs:
       with:
         node-version: '12.x'
 
-    - name: 'build'
-      run: |-
-        npm ci
-        npm run build
```

Comment:
We generally tend to keep this as we don't check in the `dist/*` and let automation build it. I don't think there is a problem leaving it, I can add the automation in a follow up.

Reply:
I removed this because it means CI would pass even if the compiled files weren't checked in. Otherwise there's no other process that enforces the TypeScript is compiled into JavaScript, right?

Reply:
@sethvargo usually once we have the automation setup, we get a PR once the TS code is merged with the compiled JS. For the PR since we compile in CI any possible compilation error is also detected in PR.

This was to prevent users from manually checking in the JS code which we cannot properly review. We can remove this and discuss offline about adding the automation.

Reply:
Hmm - that's a good point. I have an idea. I'll ping you on chat.